src/classes/DMLManager.cls

```diff
@@ -284,6 +273,50 @@ public inherited sharing class DMLManager {
 			}
 			operationRestrictedFields.put(obj.getSObjectType(),restrictedFields);
 		}
-	}	
+	}
+
+	// If the consumer desires a customized DML experience, including mock SObject unit testing:
+	// 1. Create an Apex class implementing the DMLManager.IDML interface and having the desired functionality
```

Comment:
Revise to "Create an Apex class that either extends SimpleDML and overrides the method(s) you wish to customize or implement a new Apex class that implements the full IDML interface"